sql/backup.sh

```diff
@@ -142,6 +141,20 @@ pg_rman show detail -B ${BACKUP_PATH} > ${TEST_BASE}/TEST-0004.log 2>&1
 grep -c OK ${TEST_BASE}/TEST-0004.log
 
 echo '###### BACKUP COMMAND TEST-0005 ######'
+echo '###### Make sure incremental backup work ######'
+init_catalog
+pg_rman backup -B ${BACKUP_PATH} -b full -p ${TEST_PGPORT} -d postgres --quiet;echo $?
+pg_rman validate -B ${BACKUP_PATH} --quiet
+psql -p ${TEST_PGPORT} -d postgres -c 'create table test (c1 int);'
+psql -p ${TEST_PGPORT} -d postgres -c 'insert into test values(generate_series(1,1000000));' 
+pg_rman backup -B ${BACKUP_PATH} -b incremental -p ${TEST_PGPORT} -d postgres --quiet;echo $?
+pg_rman validate -B ${BACKUP_PATH} --quiet
+pg_rman backup -B ${BACKUP_PATH} -b incremental -p ${TEST_PGPORT} -d postgres --quiet;echo $?
+pg_rman validate -B ${BACKUP_PATH} --quiet
+pg_rman show detail -B ${BACKUP_PATH} > ${TEST_BASE}/TEST-0012.log 2>&1
+grep -c 16kB ${TEST_BASE}/TEST-0012.log
```

Comment:
I think it's better to comment why you take the backups twice.
Why don't you the following comments?

```
If a differential backup was taken, the size of third backup is same as the second one and the size is bigger (XXMB).
16kB is the base backup size if nothing was changed in the database cluster.
```

But, I didn't check the second sentence is right or not.

Reply:
I think it's better to check 16KB is the size of third backup.

For example,  
```
cat ${TEST_BASE}/TEST-0012.log | tail -n 1 | awk '{print $5, $6}'
```
Though I didn't check if this works on the machine, sorry.

Reply:
Thank you for you commend.
I test that command, but it didn't get the expected results.
So I didn't change it.

Reply:
Sorry, I misunderstood the order of `pg_rman show`'s backup lists.
We need `head -n 4`.

Why don't you following? 

```
> pg_rman show detail > ~/tmp/detail.log 2>&1
> cat ~/tmp/detail.log  | head -n 4 | tail -n 1 | awk '{print $5, $6, $13}'  
INCR 16kB OK
```

If you change the argument number of `awk`, you can check arbitrary columns.